### ISSUE TITLE
Improve the default BootTidal.hs

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/Tidal#954]
It would be good to document it a bit more, switching from the old `startTidal` function to the more flexible `startStream` to make it easier for people to add new targets, etc (like in [this situation](https://club.tidalcycles.org/t/osc-out-of-tidal-ghc-network-error/4143/9)).

**Comment by ejconlon** [comment]
> Can you see any drawbacks, e.g. in terms of reduced flexibility? Would this be a breaking change for some people?

I do not believe either of those are a concern. If people want to keep their existing boot file, it will still work.

**Comment by sss-create** [comment]
closing this for now, because changes seem to be merged already and BootTidal.hs looks easier. :)
